services/movie.py

```diff
@@ -0,0 +1,33 @@
+from db.models import Movie
+from django.db.models import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(move_id: int) -> Movie:
+    return Movie.objects.get(id=move_id)
+
+
+def create_movie(movie_title: str, movie_description: str,
+                 genres_ids: list[int] = None,
+                 actors_ids: list[int] = None) -> Movie:
```

Comment:
If argument can be `None`, you should show it in type hint
```suggestion
                 genres_ids: list[int] | None = None,
                 actors_ids: list[int] | None = None) -> Movie:
```

Reply:
Fixed